server/src/main/java/org/opensearch/action/admin/indices/forcemerge/ForceMergeRequest.java

```diff
@@ -81,6 +81,8 @@ public static final class Defaults {
      */
     private final String forceMergeUUID;
 
+    private boolean shouldStoreResult;
```

Comment:
That's because this field is only used by the node which receives the api request, and it will not transported to other nodes and used by them, so serialization and deserialization for this field is not required. In other words, The node which receives the api request will generate a `parent` task and only the parent task can store its result.